CHANGELOG.md

```diff
@@ -1,3 +1,47 @@
+## [Unreleased]
+<details>
+  <summary>
+    Changes that have landed in master but are not yet released.
+    Click to see more.
+  </summary>
+
+### JS Environment Requirements
+
+ * **React 16 depends on the collection types [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) and [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set), as well as [requestAnimationFrame](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame).** If you support older browsers and devices which may not yet provide these natively (eg <IE11), consider including a global polyfill in your bundled application, such as [core-js](https://github.com/zloirock/core-js) or [babel-polyfill](https://babeljs.io/docs/usage/polyfill/).
```

Comment:
I would prefer we don't advertise `core-js` and `babel-polyfill` in this way without a specific example. I believe we had an example somewhere (in the issue?)

The trouble with this wording is it looks like we're promoting importing the whole polyfill (e.g. `import 'babel-polyfill'`). We should not. They are huge!

In the example we had, we specifically imported only the necessary ones.

Reply:
(We don't have to literally paste an example here. It's enough to link to the docs, where we might want to add the same info anyway. Or we could link to a gist.)

Reply:
Sounds good! Will fix.